### PR TITLE
Swagger addition

### PR DESCRIPTION
**Pull request by xd6d**
#23 
Done:
- Added Swagger(OpenAPI 3)
- Added swagger docs
- Updated README 
- DTO extending

### CODE REVIEW

File: src/main/java/com/qunite/api/web/mapper/QueueMapper.java [hunk 1]
```diff
@@ -12,13 +14,12 @@
...
   @Mapping(source = "creatorId", target = "creator.id")
-  Queue toEntity(QueueDto queueDto);
+  Queue toEntity(QueueCreationDto queueDto);
 
   @Mapping(source = "creator.id", target = "creatorId")
   @Mapping(target = "createdAt", dateFormat = "dd-MM-yyyy HH:mm:ss")
   QueueDto toDto(Queue queue);
 
   @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
-  @Mapping(source = "creatorId", target = "creator.id")
```
Comment (Kivanval) [inline src/main/java/com/qunite/api/web/mapper/QueueMapper.java:22]: Why did you remove that line of code?

Comment (xd6d) [reply src/main/java/com/qunite/api/web/mapper/QueueMapper.java:22]: Field "creatorId" is not present in QueueUpdateDto. In my opinion, it is wrong to have opportunity to change creator of the queue.

File: src/main/java/com/qunite/api/web/dto/user/UserUpdateDto.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import lombok.Data;
+import org.springframework.lang.Nullable;
+
+@Data
+public class UserUpdateDto {
+  @Nullable
+  private String firstName;
+  @Nullable
+  private String lastName;
```
Comment (Kivanval) [inline src/main/java/com/qunite/api/web/dto/user/UserUpdateDto.java]: Remove @Nullable

**Review by Kivanval — CHANGES_REQUESTED** [review]

**Review by Kivanval — APPROVED** [review]

**Review by vetterocki — APPROVED** [review]

**Review by NikPaukov — APPROVED** [review]
